src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -96,6 +102,19 @@ public JobCollection get(Integer offset, Integer limit, String query) {
   @Transactional
   @Override
   public org.folio.des.domain.dto.Job upsertAndSendToKafka(org.folio.des.domain.dto.Job jobDto, boolean withJobCommandSend) {
+    Optional.ofNullable(jobDto.getExportTypeSpecificParameters()).ifPresent(
+       expTypeSpecificParams -> Optional.ofNullable(expTypeSpecificParams.getVendorEdiOrdersExportConfig()).ifPresent(ediExportConfigParams->
+         Optional.ofNullable(ediExportConfigParams.getExportConfigId())
+           .ifPresent(configId -> {
+           try {
+               log.info("Looking config with id {}", configId.toString());
+               client.getConfigById(configId.toString());
+             } catch (NotFoundException e) {
+               log.info("config not found", configId.toString());
+               throw new NotFoundException(String.format(INTEGRATION_NOT_AVAILABLE, configId));
```

Comment:
What about cases when we get null in the chain "ifPresent"s? Should code throw exception?

Reply:
i dont think we will get null . 
i think if we are getting null that means job is not available. @SerhiiNosko please suggest.